packages/conduit-mirage/conduit-mirage.2.0.1+dune/opam

```diff
@@ -1,34 +1,44 @@
 opam-version: "2.0"
-version: "3.1.0"
 maintainer: "anil@recoil.org"
 authors: ["Anil Madhavapeddy" "Thomas Leonard" "Thomas Gazagnaire"]
 license: "ISC"
 tags: "org:mirage"
 homepage: "https://github.com/mirage/ocaml-conduit"
-bug-reports: "https://github.com/dune-universe/ocaml-conduit/issues"
+bug-reports: "https://github.com/mirage/ocaml-conduit/issues"
 depends: [
   "ocaml" {>= "4.03.0"}
   "dune"
-  "ppx_sexp_conv" {< "v0.13"}
-  "sexplib" {< "v0.13"}
-  "ipaddr" {>="3.0.0"}
+  "ppx_sexp_conv" {>="v0.9.0"}
```

Comment:
Any particular reason to remove the upper bound here or is it an upstream change?

Not that it matters too much but since JS packages tend to break in between minor changes I think it made some sense to have it to prevent breakage on the next batch of JS release.